components/ArtPiecePreview/index.js

```diff
@@ -0,0 +1,13 @@
+export default function ArtPiecePreview({ imageSource, title, artist }) {
+  return (
+    <div>
+      <img
```

Comment:
"Use the next/image component to display the image" - please use the Image component - provided by Next.